repositories.yaml

```diff
@@ -1083,3 +1083,26 @@ applications:
       - v1_name: rally-attention-stream
         app_id: rally-attention-stream
     skip_documentation: true
+
+  - app_name: viu_politica
+    canonical_app_name: Viu Politica
+    app_description: |
+      This is a "spin-off" of the Mozilla YouTube RegretsReporter extension.
+      It is an opt-in browser extension where users can anonymously tag videos
+      related to the 2022 Brazilian Elections.
+    url: https://github.com/evoluchico/viu-politica
```

Comment:
@evoluchico, are you planning on migrating this to the Mozilla github account?

Reply:
Our initial plan was to just host the code on my github account. I'm okay with moving it to something like https://github.com/mozilla/ or https://github.com/mozilla-extensions/ if it is necessary, but if it isn't, would it be okay to keep it in my github account?

Reply:
We talked this over Slack. The repo is going to migrate to Mozilla.